packages/web-app-external/src/App.vue

```diff
@@ -36,24 +36,9 @@
 import { mapGetters } from 'vuex'
 import ErrorScreen from './components/ErrorScreen.vue'
 import LoadingScreen from './components/LoadingScreen.vue'
-
-// FIXME: hacky, get rid asap, just a workaround
-// same as packages/web-app-files/src/views/PublicFiles.vue
-const unauthenticatedUserReady = async (router, store) => {
-  if (store.getters.userReady) {
-    return
-  }
-
-  const publicToken = router.currentRoute.query['public-token']
-  const publicLinkPassword = store.getters['Files/publicLinkPassword']
-
-  await store.dispatch('loadCapabilities', {
-    publicToken,
-    ...(publicLinkPassword && { user: 'public', password: publicLinkPassword })
-  })
-
-  store.commit('SET_USER_READY', true)
```

Comment:
Loading the capabilities and setting the user ready is important. Wondering how you still have the app working at all without it...

Reply:
You are totally right, sorry. I didn't realize that it only worked because this was cached.